index.js

```diff
@@ -24,10 +24,12 @@ module.exports = function buildAirbnbPreset(context, options) {
   var transpileTargets = (options && options.targets) ||
     buildTargets(options || {});
 
+  var debug = (options && typeof options.debug === 'boolean') ? !!options.debug : false;
```

Comment:
If type of `options.debug` is boolean, it is redundant to double negate it, isn't it?